includes/class-wc-payments-payment-request-button-handler.php

```diff
@@ -101,175 +101,6 @@ public function init() {
 		// will be used to calculate it whenever the option value is retrieved instead.
 		// It's used for displaying inbox notifications.
 		add_filter( 'pre_option_wcpay_is_apple_pay_enabled', [ $this, 'get_option_is_apple_pay_enabled' ], 10, 1 );
-
-		add_action(
```

Comment:
All moved to `includes/express-checkout/class-wc-payments-express-checkout-ajax-handler.php`